client/src/components/AlumniMentorship.js

```diff
@@ -540,6 +553,19 @@ class RequestCards extends Component {
         }
     }
 
+    toggleActionItemModal(e) {
+        let requestDetails = this.props.requests.find(request => request._id === e.currentTarget.getAttribute('requestid'))
```

Comment:
We can remove everything from here except
`this.setState({showActionItemsModal: !this.state.showActionItemsModal })`
There is no actionItems state we need to store